com.asgardex.asgardex-desktop.yml

```
@@ -0,0 +1,34 @@
+id: com.asgardex.asgardex-desktop
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
```

```suggestion
  - org.freedesktop.Sdk.Extension.node20
build-options:
  append-path: /usr/lib/sdk/node20/bin
```

Without this, `yarn` won't exist in the PATH during the build.